app/models/concerns/requestable.rb

```diff
@@ -16,4 +16,12 @@ module Requestable
       update!(state: "received", received_at: Time.zone.now)
     end
   end
+
+  def expires_after
+    6.weeks
```

Comment:
Is 6 weeks the default expiry period? (ie. if not one of the countries which expires after 4 weeks in `FurtherInformationRequest`)
Perhaps it could be shared via a constant or method if so. 
Not a blocker.

Reply:
I'm actually thinking I might take this method out of `Requestable` and define it explicitly on `FurtherInformationRequest` and `ReferenceRequest`, since it's probably not the responsibility of the concern to have a default.